src/encoded/static/scss/encoded/_base.scss

```diff
@@ -165,6 +192,7 @@ blockquote {
 $disabled-color-factor: 20%;
 
 .btn {
+    font-family: Quicksand;
```

Comment:
Probably should have added a fallback, but should be OK.

Reply:
I missed this. The fallback I believe defaults to Times New Roman. Is that acceptable?

Reply:
I think just “sans-serif” as the default should be fine.